src/server/game/Chat/Chat.h

```diff
@@ -48,6 +59,13 @@ class AC_GAME_API ChatHandler
     // Builds chat packet and returns receiver guid position in the packet to substitute in whisper builders
     static size_t BuildChatPacket(WorldPacket& data, ChatMsg chatType, Language language, WorldObject const* sender, WorldObject const* receiver, std::string_view message, uint32 achievementId = 0, std::string const& channelName = "", LocaleConstant locale = DEFAULT_LOCALE);
 
+    // All in one chat message builder
+    static void BuildChatPacket(
+            WorldPacket& data, ChatMsg msgtype, std::string_view message, Language language = LANG_UNIVERSAL, PlayerChatTag chatTag = CHAT_TAG_NONE,
+            ObjectGuid const& senderGuid = ObjectGuid(), std::string_view senderName = nullptr,
```

Comment:
/home/ubuntu22/azerothcore-wotlk/src/server/game/Chat/Chat.h:66:88: warning: null passed to a callee that requires a non-null argument [-Wnonnull]
            ObjectGuid const& senderGuid = ObjectGuid(), std::string_view senderName = nullptr,
                                                                                       ^~~~~~~
/home/ubuntu22/azerothcore-wotlk/src/server/game/Chat/Chat.h:67:88: warning: null passed to a callee that requires a non-null argument [-Wnonnull]
            ObjectGuid const& targetGuid = ObjectGuid(), std::string_view targetName = nullptr,

Reply:
Please fix this warning

Reply:
`std::string_view senderName = {}` should be okay